runtime/compiler/runtime/RelocationRuntime.cpp

```diff
@@ -1120,7 +1120,18 @@ TR_SharedCacheRelocationRuntime::allocateSpaceInCodeCache(UDATA codeSize)
       }
 
    uint8_t *coldCode;
-   U_8 *codeStart = manager->allocateCodeMemory(codeSize, 0, &_codeCache, &coldCode, false);
+   U_8 *codeStart;
+   bool installIntoCold = TR::Options::getCmdLineOptions()->getOption(TR_ReinstallAOTToColdCode);
```

Comment:
Why is the option called "reinstall"? Why not "install", especially that the variable is called `installIntoCold`?

Reply:
I think local variables can have somewhat shortened names. The option is already merged into OMR. Do you think it's worth renaming it? @mpirvu

Reply:
You can open an issue to follow up, but in the interest of delivering this quicker it can stay like this for the time being.